src/transaction/lock_manager.c

```diff
@@ -3970,15 +3961,8 @@ lock_internal_perform_unlock_object (THREAD_ENTRY * thread_p, LK_ENTRY * entry_p
   int rv;
 
 #if defined(LK_DUMP)
-  if (lk_Gl.dump_level >= 1)
-    {
-      fprintf (stderr,
-	       "LK_DUMP::lk_internal_unlock_object()\n"
-	       "  tran(%2d) : oid(%2d|%3d|%3d), class_oid(%2d|%3d|%3d), LOCK(%7s)\n", entry_ptr->tran_index,
-	       entry_ptr->res_head->oid.volid, entry_ptr->res_head->oid.pageid, entry_ptr->res_head->oid.slotid,
-	       entry_ptr->res_head->class_oid.volid, entry_ptr->res_head->class_oid.pageid,
-	       entry_ptr->res_head->class_oid.slotid, LOCK_TO_LOCKMODE_STRING (entry_ptr->granted_mode));
-    }
+  lock_dump_internal_lock_action (__func__, entry_ptr->tran_index, &entry_ptr->res_head->key.oid,
```

Comment:
What is the meaning of the -42?

Reply:
Invalid.
Some negative values already have meaning:
```
enum
{
/* Value to timeout immediately.. not wait */
  LK_ZERO_WAIT = 0,
/* Value to wait forever */
  LK_INFINITE_WAIT = -1,
/* Value to force a timeout without setting errors */
  LK_FORCE_ZERO_WAIT = -2
};
```
All positive values also have meaning.
`-42` looked like a safe bet to signal an invalid value for timeout

Reply:
I don't yet know the purpose of logging wait_msecs in the lock_dump_internal_lock_action() function, but rather than hardcoding -42, how about the following?

```
enum
{
/* Value to timeout immediately.. not wait */
  LK_ZERO_WAIT = 0,
/* Value to wait forever */
  LK_INFINITE_WAIT = -1,
/* Value to force a timeout without setting errors */
  LK_FORCE_ZERO_WAIT = -2
#if defined(LK_DUMP)
/* Some description ... */
  LK_INVAILD_WAIT_TIME = -42
#endif
};

Reply:
I removed msec logging.